INSTALL.md

```diff
@@ -1,204 +1,67 @@
-```bash
-echo 'source /Users/[YOUR MAC OSX USERNAME]/homebrew/opt/autoenv/activate.sh' >> ~/.bash_profile
-```
-
-If you need to find this info again later, you can run:
-```bash
-brew info autoenv
-```
-
-> **NOTE:** If you use ZSH you’ll need to use [zsh-autoenv](https://github.com/Tarrasch/zsh-autoenv),
-  but we can’t provide support for issues that may arise.
-
-### Elasticsearch
-
-[Install Elasticsearch](http://www.elastic.co/guide/en/elasticsearch/reference/current/setup.html)
-however you’d like. (We use [Homebrew](http://brew.sh)):
-```bash
-brew install elasticsearch
-```
-
-Just as with autoenv, Homebrew will output similar instructions after installation:
-```bash
-To have launchd start elasticsearch at login:
-    ln -sfv /Users/[YOUR MAC OSX USERNAME]/homebrew/opt/elasticsearch/*.plist ~/Library/LaunchAgents
-Then to load elasticsearch now:
-    launchctl load ~/Library/LaunchAgents/homebrew.mxcl.elasticsearch.plist
-Or, if you don’t want/need launchctl, you can just run:
-    elasticsearch --config=/Users/[YOUR MAC OSX USERNAME]/homebrew/opt/elasticsearch/config/elasticsearch.yml
-```
-
-Any time you work on the project, you’ll need to open a new tab and run that last line.
-If you’ll be working on the project consistently, we suggest using the first option
-utilizing `launchd`.
-
-If you need to find this info again later, you can run:
-```bash
-brew info elasticsearch
-```
-
-### MYSQL Database
-Start MYSQL with the following command:
-```
-mysql.server start
-```
-
-Then run the MYSQL creation script from project root directory:
-```
-./create-mysql-db.sh
-```
-
-If you would like to have a custom database setup
-then you can pass in the necessary arguments:
-
-```
-./create-mysql-db.sh <dbname> <username> <password>
-```
-
-> **NOTE:** Be sure to update your local settings in
-  `cfgov/cfgov/settings/local.py` to account for these changes.
-
-
-If something goes wrong and you'd like to delete the database
-and start again, you can do so with:
-
-```
-mysql v1 -u root -p -e 'drop database v1;'
-```
+The project uses a number of environment variables.
+The `setup.sh` script will create a `.env` file for you
+from the `.env_SAMPLE` file found in the repository,
+if you don't already have one.
 
-### Virtual Environment
-In the project root directory,
-create a virtualenv that you’ll name `cfgov-refresh`:
-```bash
-mkvirtualenv cfgov-refresh
-```
+Inside the `.env` file you can customize the project environment configuration.
 
-The new virtualenv will activate right away. To activate it later on
-(say, in a new terminal session) use the command `workon cfgov-refresh`.
-You’ll know you have a virtual environment activated if you see the name of it in
-parentheses before your terminal prompt. Ex:
+If you would like to manually copy the environment settings,
+copy the `.env_SAMPLE` file and un-comment each variable after
+adding your own values.
 ```bash
-(cfgov-refresh)$
+cp -a .env_SAMPLE .env && open .env
```

Comment:
You might move `open .env` to another line and note that if an editor isn't set this may do weird things like open the file as a script in a new Terminal window.